.github/workflows/ts-build.yml

```diff
@@ -0,0 +1,30 @@
+on:
+  workflow_call: {}
+
+env:
+  TURBO_CACHE_DIR: ./.turbo/cache
+  TURBO_TELEMETRY_DISABLED: true
+  TURBO_RUN_SUMMARY: true
+
+defaults:
+  run:
+    working-directory: ./typescript
+
+jobs:
+  run:
+    name: Build
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@d632683dd7b4114ad314bca15554477dd762a938
+
+      - name: Setup
+        uses: ./.github/actions/ts-setup
+        with:
+          package-manager: yarn
+          working-directory: ./typescript
+
+      - name: Build
```

Comment:
if we always expect to use `yarn`, what do you think about just going for `yarn` everywhere?

i.e. Maybe we can avoid taking options to start with in `ts-setup` and "just" standardize on `yarn` for now?